data/strings/types_strings.txt

```diff
@@ -6833,7 +6833,7 @@
     ar = مطبخ إيطالي؛بيتزا
     cs = Italská, pizza
     da = Italiensk, pizza
-    de = Italienische Küche; Pizza
+    de = Italienisch; Pizza
```

Comment:
`type.cuisine.italian_pizza`
Two points here:
1. I think all current translations are a bit off here. It should be something like "Italian-style pizza" or just "Italian pizza".
2. Its just 1.4k uses!
While e.g. `italian;pizza` has 3k, `pizza;italian` has 2k uses, but I guess compound semicolon delimited values are not parsed by OM? @vng 
I would rather not spend time improving this translation and  merge `italian_pizza` into just `pizza` (98k uses).